com.sap.cloud.lm.sl.cf.client/src/test/java/com/sap/cloud/lm/sl/cf/client/util/TokenFactoryTest.java

```diff
@@ -99,11 +103,61 @@ public void testCreateToken() {
 
     }
 
+    @RunWith(Parameterized.class)
+    public static class ExchangedTokenFactoryTest {

```

Comment:
I imagine this test to be as part of XS project or renamed to be more general for all CF use cases.